pkgs/applications/misc/mediaelch/default.nix

```diff
@@ -28,20 +31,38 @@ mkDerivation rec {
     fetchSubmodules = true;
   };
 
-  nativeBuildInputs = [ qmake qttools ];
-
-  buildInputs = [ curl ffmpeg libmediainfo libzen qtbase qtdeclarative qtmultimedia qtsvg ];
+  nativeBuildInputs = [
+    qmake
+    qttools
+    wrapQtAppsHook
+  ];
 
-  qmakeFlags = [
-    "USE_EXTERN_QUAZIP=${quazip}/include/quazip5"
+  buildInputs = [
+    curl
+    ffmpeg
+    libmediainfo
+    libzen
+    qt5compat
+    qtbase
+    qtdeclarative
+    qtmultimedia
+    qtsvg
+    qtwayland
   ];
 
   postPatch = ''
-    substituteInPlace MediaElch.pro --replace "/usr" "$out"
+    substituteInPlace MediaElch.pro \
+      --replace "/usr" "$out" \
+      --replace "lrelease" "${lib.getDev qttools}/bin/lrelease"
```

Comment:
It should have been fixed in https://github.com/NixOS/nixpkgs/issues/202293

Reply:
thats exactly the cause, why I pinged you both. But If I execute it without the patching `lrelease`:
```
/nix/store/wlgj12rfhsrm5a0mixk2fsjgzcahykzd-qtbase-6.4.1-dev/libexec/rcc -name ui ui.qrc -o qrc_ui.cpp
/nix/store/wlgj12rfhsrm5a0mixk2fsjgzcahykzd-qtbase-6.4.1-dev/bin/lrelease data/i18n/MediaElch_cs_CZ.ts -qm .qm/MediaElch_cs_CZ.qm
/nix/store/dsd5gz46hdbdk2rfdimqddhq6m8m8fqs-bash-5.1-p16/bin/bash: line 1: /nix/store/wlgj12rfhsrm5a0mixk2fsjgzcahykzd-qtbase-6.4.1-dev/bin/lrelease: No such file or directory
make: *** [Makefile:2529: .qm/MediaElch_cs_CZ.qm] Error 127
```

Reply:
I noticed that mediaelch have an alternative cmake build system available, may we try that instead of qmake? Our qt6 qmake is sadly in quite a poor shape.

Reply:
My take on it: https://github.com/NickCao/nixpkgs/commit/cfa04447a64919dcb463ee515882bdabd2992a58

Reply:
I think `qttools` can also be placed in the `nativeBuildInputs`. Other then that your take looks very good and works as expected. I applied the changes in this MR as well.

Reply:
Thanks for your advice, help and suggestions.